pkgs/applications/networking/remote/nice-dcv-client/default.nix

```diff
@@ -0,0 +1,89 @@
+{ stdenv
+, fetchurl
+, glib
+, libX11
+, gst_all_1
+, sqlite
+, epoxy
+, pango
+, cairo
+, gdk-pixbuf
+, e2fsprogs
+, libkrb5
+, libva
+, openssl
+, pcsclite
+, gtk3
+, libselinux
+, libxml2
+, python3Packages
+, cpio
+, autoPatchelfHook
+}:
+
+stdenv.mkDerivation rec {
+  pname = "nice-dcv-client";
+  version = "2020.2.1737-1";
+
+  src =
+    fetchurl {
+      url = "https://d1uj6qtbmh3dt5.cloudfront.net/2020.2/Clients/nice-dcv-viewer-${version}.el8.x86_64.rpm";
+      sha256 = "sha256-SUpfHd/Btc07cfjc3zx5I5BiNatr/c4E2/mfJuU4R1E=";
+    };
+
+  nativeBuildInputs = [ autoPatchelfHook python3Packages.rpm ];
+  unpackPhase = ''
+    rpm2cpio $src | ${cpio}/bin/cpio -idm
+    echo "pwd = $(pwd)"
```

Comment:
This looks like some left over debugging stuff?

Reply:
yep, fixed and force-pushed.